airgun/widgets.py

```diff
@@ -2090,3 +2091,48 @@ class Link(Text):
     def fill(self, value):
         if value:
             self.browser.click(self)
+
+
+class PopOverWidget(GenericLocatorWidget):
```

Comment:
1) Don't think that we need to put in basic widget module as it specific to settings module only
2) Can we think about better name?

Reply:
1. One point 1, tomorrow it could have similar widget then? (All the actions inside that widget are specifically are common like widget nothing related settings module ) 
2. What would you think, give me the suggested name?